### DISCUSSION TITLE
Project: Dashboard thể hiện số liệu của ông dev ở mạng xã hội

### DISCUSSION THREAD

**Discussion by cutbypham** [OngDev/community#2] [Project Ideas]
## design review

![Desktop design](https://res.cloudinary.com/dz209s6jk/image/upload/v1585840948/Challenges/ftdvklbtn6y7ydjxjjuj.jpg)

![mobile design](https://res.cloudinary.com/dz209s6jk/image/upload/v1585840948/Challenges/iwqqeasudxd1i331actz.jpg)

Resource: JPEG design files for mobile & desktop layouts, Style guide for fonts, colors, etc, Optimized image assets, README file to help you get started, HTML file with pre-written content

Ref:
https://www.frontendmentor.io/challenges/social-media-dashboard-with-theme-switcher-6oY8ozp_H

**Comment by NamNguyen2110** [comment]
Backend Spring Boot + VueJS

**Reply by khanhhh27** [reply]
dùng js thuần đi anh ^^

**Reply by dongnguyenvn** [reply]
Node.js + React

**Reply by NamNguyen2110** [reply]
Viết bằng VueJS để có độ so sánh vs ReactJS bạn ạ. Còn spring boot muốn xem nó tương tác với API bên ngoài như nào... chứ những cái về IO thì nodejs nó sẽ nhanh hơn....nhưng làm cái gì khác khác nó hay hơn mà

**Comment by andynguyenit78x** [comment]
NodeJS - ReactJS

**Comment by cutbypham** [comment]
viết html, css, js chay tịnh

**Comment by vietun9** [comment]
js thuần frontend 
backend express

**Comment by cutbypham** [comment]
https://ongdev.com/
